app/email-incoming/index.ts

```diff
@@ -160,3 +165,11 @@ async function getLegacyUserData(
     }
   )
 }
+
+function validateMessageIsNotSpam(messageReceipt: any) {
+  if (
+    messageReceipt.spamVerdict.status != 'PASS' ||
+    messageReceipt.virusVerdict.status != 'PASS'
+  )
+    throw new Error('Message caught in virus/spam detection.')
+}
```

Comment:
I would prefer to keep this inline, inside the `handleRecord` function, so that we don't need the `any` type.

Reply:
Can do! Will have that and the other verdicts included in the next commit